jest.setup.js

```diff
@@ -48,8 +49,20 @@ jest.mock("electron", () => {
 });
 
 beforeAll(async () => {
+	migrateProfileFixtures(env);
+
 	await env.verify();
 	await env.boot();
+
+	await Promise.allSettled(
```

Comment:
Using `allSettled` here can result in the migration failing and you not finding out about it unless you explicitly loop over each promise and check for the resolution status.

Reply:
@faustbrian fixed.